bucket/vmware-workstation-player-np.json

```diff
@@ -0,0 +1,47 @@
+{
+    "version": "17.0.0-20800274",
+    "homepage": "https://www.vmware.com/ca/products/workstation-player.html",
+    "description": "An application for creating and running virtual machines on your computer.",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://www.vmware.com/ca/help/legal.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://download3.vmware.com/software/WKST-PLAYER-1700/VMware-player-full-17.0.0-20800274.exe#/setup.exe",
+            "hash": "ae238c457a100f15a6d009bfeee4ff277dce181e96a279d486f07f30da433183"
+        }
+    },
+    "pre_install": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "Start-Process \"$dir\\setup.exe\" -Wait -Verb 'RunAs' -ArgumentList '/x'; Remove-Item \"$dir\\setup.exe\"",
+        "$msiInstaller = (Get-ChildItem \"$env:TEMP\" -Filter 'VMware*' -Recurse -Force).FullName; Copy-Item \"$msiInstaller\" \"$dir\\setup.msi\" -Force",
+        "Start-Process 'msiexec' -Wait -Verb 'RunAs' -ArgumentList @('/i', \"$dir\\setup.msi\", '/qn', '/norestart', 'REBOOT=ReallySuppress', 'EULAS_AGREED=1', 'ADDLOCAL=ALL')",
+        "Remove-Item \"$msiInstaller\" -Force"
+    ],
+    "pre_uninstall": [
+        "if ($cmd -eq 'uninstall') {",
+        "   if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "   Start-Process 'msiexec' -Wait -Verb 'RunAs' -ArgumentList @('/x', \"$dir\\setup.msi\", '/qn', '/norestart', 'REBOOT=ReallySuppress')",
+        "   Start-Sleep -Seconds 2",
+        "   Write-Host \"Please restart your computer to uninstall $app properly\" -F 'Red'",
+        "}"
+    ],
+    "checkver": {
+        "url": "https://scoop.sh",
+        "useragent": "$app/$version",
+        "script": [
+            "$redirUrl = [System.Net.HttpWebRequest]::Create('https://www.vmware.com/go/getplayer-win').GetResponse().ResponseUri.AbsoluteUri",
+            "return \"$redirUrl\""
+        ],
+        "regex": "WKST-PLAYER-(<CleanVer>[\\d]+)\\/VMware-player-full-([\\d.]+-[\\w]+)",

```

Comment:
@Zliced13 I think a question mark needs to be added:
```
WKST-PLAYER-(?<CleanVer>[\\d]+)\\/VMware-player-full-([\\d.]+-[\\w]+)
```

Reply:
I'll try this later on today.